File: akka-samples/akka-sample-cluster-java/src/main/java/sample/cluster/factorial/FactorialApp.java
package sample.cluster.factorial;

public class FactorialApp {

  public static void main(String[] args) {
    // starting 3 backend nodes and 1 frontend node
    FactorialBackendMain.main(new String[] { "2551" });
    FactorialBackendMain.main(new String[] { "2552" });
    FactorialBackendMain.main(new String[0]);
    FactorialFrontendMain.main(new String[0]);
  }
}

File: akka-samples/akka-sample-cluster-scala/src/main/scala/sample/cluster/factorial/FactorialApp.scala
package sample.cluster.factorial

object FactorialApp {
  def main(args: Array[String]): Unit = {
    // starting 3 backend nodes and 1 frontend node
    FactorialBackend.main(Seq("2551").toArray)
    FactorialBackend.main(Seq("2552").toArray)
    FactorialBackend.main(Array.empty)
    FactorialFrontend.main(Array.empty)
  }
}